pyemma/coordinates/data/writer.py

```diff
@@ -26,27 +26,25 @@
 from __future__ import absolute_import
 
 import numpy as np
-from pyemma.coordinates.transform.transformer import Transformer
+from pyemma.coordinates.transform.transformer import StreamingTransformer
 
 
-class WriterCSV(Transformer):
+class WriterCSV(StreamingTransformer):
 
     '''
-    shall write to csv files
+    write to csv files via numpy
```

Comment:
move this stuff to a public method in DataSource. Eg. "write_to_csv(filename,...)"